docs/releases/3.0.8.txt

```diff
@@ -18,3 +18,5 @@ Bugfixes
 * Reallowed, following a regression in Django 3.0, non-expressions having a
   ``filterable`` attribute to be used as the right-hand side in queryset
   filters (:ticket:`31664`).
+
+* Fixed a crash in ``django-admin runserver`` on Windows (:ticket:`31716`).
```

Comment:
I'm not sure if this qualifies for a backport. This wouldn't have worked for many an age no? 🤔